src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```
@@ -827,9 +937,53 @@ internal void HandleWorkerMetadataRequestError(Exception exc)
             _eventManager.Publish(new WorkerErrorEvent(_runtime, Id, exc));
         }
 
+        private ValueTask SendStreamingMessageAsync(StreamingMessage msg)
+        {
+            if (_outbound is null)
+            {
+                return default;
+            }
+            var evt = new OutboundGrpcEvent(_workerId, msg);
+            return _outbound.TryWrite(evt) ? default : _outbound.WriteAsync(evt);
+        }
+
         private void SendStreamingMessage(StreamingMessage msg)
         {
-            _eventManager.Publish(new OutboundGrpcEvent(_workerId, msg));
+            if (_outbound is null)
+            {
+                return;
+            }
+            var evt = new OutboundGrpcEvent(_workerId, msg);
+            if (!_outbound.TryWrite(evt))
+            {
+                var pending = _outbound.WriteAsync(evt);
+                if (pending.IsCompleted)
+                {
+                    try
+                    {
+                        pending.GetAwaiter().GetResult(); // ensure observed, for IVTS reasons
+                    }
+                    catch
+                    {
+                        // suppress failure
```

Just wondering why we want to suppress the failure here